external_deps/build.sh

```diff
@@ -623,13 +623,10 @@ build_install() {
 build_package() {
 	cd "${WORK_DIR}"
 	case "${PLATFORM}" in
-	mingw*|msvc*)
-		rm -f "${PLATFORM}-${DEPS_VERSION}.zip"
-		zip -r "${PLATFORM}-${DEPS_VERSION}.zip" "${PLATFORM}-${DEPS_VERSION}"
-		;;
 	*)
-		rm -f "${PLATFORM}-${DEPS_VERSION}.tar.bz2"
-		tar cvjf "${PLATFORM}-${DEPS_VERSION}.tar.bz2" "${PLATFORM}-${DEPS_VERSION}"
+		export XZ_OPT='-9'
```

Comment:
Would rather see `XZ_OPT=-9 tar cvJf`... than set a variable that persists for the rest of the program.